src/app/components/elements/UserList.jsx

```diff
@@ -23,18 +23,19 @@ class UserList extends React.Component {
 
     render() {
         const { state: { historyIndex } } = this;
+        const account = this.props.account;
         const users = this.props.users;
         const title = this.props.title;
 
         let idx = 0;
         let user_list = users.map(user => (
-            <UserListRow user={user} key={idx++} />
+            <UserListRow account={account} user={user} key={idx++} />
         ));
+        user_list = user_list.toArray();
 
         let currentIndex = -1;
-        const usersLength = users.length;
+        const usersLength = users.size;
         const limitedIndex = Math.min(historyIndex, usersLength - PER_PAGE);
-
         user_list = user_list.reverse().filter(() => {
```

Comment:
ahhh it's being redefined again down here...ok maybe we should split off this refactor stuff into another issue 🙃

Reply:
Yeah so this bug was introduced by  me not understanding that users was an immutable type.